<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_brightness_contrast</class>
 <widget class="QDialog" name="dialog_brightness_contrast">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Brightness and Contrast</string>
  </property>
  <widget class="QDialogButtonBox" name="btnbox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSlider" name="hslider_brightness">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_brightness">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Brightness</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_brightness_value">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_contrast">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Contrast</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_contrast_value">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QSlider" name="hslider_contrast">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="sliderPosition">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnbox</sender>
   <signal>accepted()</signal>
   <receiver>dialog_brightness_contrast</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnbox</sender>
   <signal>rejected()</signal>
   <receiver>dialog_brightness_contrast</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hslider_brightness</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lbl_brightness_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
